cmd/gosec/main.go

```diff
@@ -206,6 +209,14 @@ func saveOutput(filename, format string, color bool, paths []string, issues []*g
 		if err != nil {
 			return err
 		}
+		if stdout {
+			format = "text"
```

Comment:
IMHO format shouldn't be changed when `stdout` shall only means a redirect of the stream.